src/main/cpp/main.cpp

```diff
@@ -397,22 +397,20 @@ int main()
 				// currentPlayer->goToJail();
 				break;
 			case HOUSE: //Change it to where if you can't pay rent then you either sell a property to the owner or go bankrupt(game over)
-				// game.landOnHouseTile();
-				break;
-			case CHEST:
-				break;
-			case FREE:
-				break;
-			case RAILROAD:
-				break;
-			case CHANCE:
+				game.landOnHouseTile();
 				break;
 			case TAX:
 				game.print("You have to pay $75!", true, false);
 				break;
 			case LUXURYTAX:
 				game.landOnLuxuryTaxTile();
 				break;
+			//All other cases do nothing

```

Comment:
Some of the implementation has not yet been added in to different tile types. The GO and FREE tiles don't do anything. I would move them back to how they were for now.

Reply:
Done